src/components/article-body/body-components/table.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentFactory } from '../component-factory';
+import theme from '@mdb/flora/theme';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+const headerCellStyles = (align: any, index: number) => {
+    const alignment = align[index];
+    return {
+        paddingTop: ['inc20', 'inc40', 'inc40', 'inc40'],
+        paddingBottom: ['inc20', 'inc40', 'inc40', 'inc40'],
+        paddingLeft: ['inc20', 'inc30', 'inc30', 'inc30'],
+        paddingRight: ['inc20', 'inc30', 'inc30', 'inc30'],
+        textAlign: alignment,
+        whiteSpace: 'nowrap' as 'nowrap',
+    };
+};
+
+const headerRowStyles = () => {
+    return {
+        borderBottom: `solid ${theme.borderWidths.inc30}`,
+        borderBottomColor: theme.colors.blue80,
+    };
+};
+
+const getNestedValue = (p: any, o: any) => {
```

Comment:
Can we be more explicit in describing what `p` and `o` are here when coming up with param names? also, assign correct type and not any